tuna/fin_eval.py

```diff
@@ -179,45 +180,50 @@ def get_fin_input(self):
     return self.machine.write_file(json.dumps(fjob, indent=2).encode(),
                                    is_temp=True)
 
+  def update_fdb_eval_entry(self, session, fdb_obj):
+    """update fdb with individual fin json entry"""
+    if fdb_obj['evaluated']:
+      obj, _ = self.get_fdb_entry(session,
+                                  self.solver_id_map[fdb_obj['solver_name']])
+      if not obj:
+        self.logger.info(
+            'Unable to find fdb entry for config: %s, solver: %s, '\
+            'arch: %s, num_cu: %s, direction: %s',
+            self.config.id, self.solver_id_map[fdb_obj['solver_name']],
+            self.dbt.session.arch, self.dbt.session.num_cu, self.config.direction)
+        raise ValueError("Unable to query find db entry")
+      fdb_entry = obj
+      fdb_entry.alg_lib = fdb_obj['algorithm']
+      fdb_entry.kernel_time = fdb_obj['time']
+      fdb_entry.workspace_sz = fdb_obj['workspace']
+      fdb_entry.session = self.dbt.session.id
+      fdb_entry.params = fdb_obj['params']
```

Comment:
I am current using 'staging' branch for tuning. In 'staging' branch, we suppose that the fdb table does not have column 'params'. My concern is that whether we could merge this branch to 'staging' without the 'params', so that we could do tuning without altering the fdb table.

Reply:
The ixt-rack-15 db already has params in the fdb table. so this shouldn't cause an issue.